### ISSUE TITLE
Possible error, cannot resolve pic in webpack 4

### ISSUE THREAD

**Issue by Connorelsea** [brunocodutra/webapp-webpack-plugin#25]
When using webpack 4 and the latest version of this library, I am getting this error, although the logo.png file does exist in that directory.

`EntryModuleNotFoundError: Entry module not found: Error: Can't resolve 'logo.png' in '/Users/connorelsea/Projects/remixplatform'`

![image](https://user-images.githubusercontent.com/1844458/37316252-01078e46-262c-11e8-8e2f-41fb20363246.png)

This is the webpack config I am using. https://gist.github.com/Connorelsea/6b7ed5e7fd0e4ab83d839e3e8c698701

**Comment by Connorelsea** [comment]
Update, I got it to work by changing `logo.png` to `./logo.png`

**Comment by brunocodutra** [comment]
### TL;DR;

Despite the somewhat misleading error message you get, Webpack actually tries to resolve `logo.png` within `node_modules` . 

To fix it you can either:

* use an absolute path, i.e. `path.join(__dirname, "logo.png)` or `path.resolve(__dirname, "logo.svg")`,
* use a relative path, but keep in mind that this relies on the path where you run `npm` unless you set `context` to something more meaningful like `__dirname`,
* or add `resolve { modules: [__dirname, "node_modules"] }` to your Webpack config.

In your specific case, I suggest the first option, but any should do really.

### In Detail 

There are three cases to consider:

* If  `/path/to/logo` is absolute, there is nothing to resolve, the path is just used as is.

* If `./path/to/logo` is relative, it's resolved with respect to Webpack's [`context`](https://webpack.js.org/configuration/entry-context/#context), which defaults to `process.cwd()`.

* If `path/to/logo` is neither explicitly relative nor absolute, Webpack attempts to resolve it according to [`resolve.modules`](https://webpack.js.org/configuration/resolve/#resolve-modules), which defaults to `modules: ["node_modules"]`.

If you don't mind, I'll keep this open to remind myself to elaborate on that in the README. Either way, thanks for bringing my attention to the fact this is not clear.
